.github/workflows/ci.yml

```diff
@@ -1,70 +1,174 @@
 name: CI
 on:
-  push:
-    branches:
-      - pangolin
-      - master
-  pull_request:
-    branches:
-      - pangolin
-      - master
-
-env:
-  CARGO_TERM_COLOR: always
+    push:
+        branches:
+            - main
+            - master
+    pull_request:
+        branches:
+            - main
+            - master
 
 jobs:
-  check-default-features:
-    name: Check `default-features`
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-      - uses: actions-rs/toolchain@v1
-        with:
-          toolchain: nightly
-      - name: Build checker
-        run: cargo b --manifest-path .maintain/check-default-features/Cargo.toml
-      - name: Run checker
-        run: .maintain/check-default-features/target/debug/check-default-features
-
-  test:
-    name: Run Tests
-    runs-on: ubuntu-latest
-    strategy:
-      fail-fast: false
-    steps:
-      - uses: actions/checkout@v2
-
-      - uses: actions-rs/toolchain@v1
-        with:
-          toolchain: nightly-2021-03-01
-          override: true
-          target: wasm32-unknown-unknown
-
-      - name: Overall test
-        run: cargo t
-
-  dvm-rpc-test:
-    name: Run DVM RPC Tests
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-
-      - uses: actions-rs/toolchain@v1
-        with:
-          toolchain: nightly-2021-03-01
-          override: true
-          target: wasm32-unknown-unknown
-
-      - name: Build and Run node
-        run: |
-          cargo b
-          ./target/debug/drml --dev --tmp &
-          sleep 180
-
-      - name: Set up node env
-        uses: actions/setup-node@v1
-        with:
-          node-version: 10
-
-      - name: Run dvm tests
-        run: cd tests/dvm && npm install && npm test
+    build_kit:
+        name: Build kit
+        runs-on: ubuntu-latest
+        steps:
+            - name: Fetch latest code
+              uses: actions/checkout@v2
+              with:
+                  repository: darwinia-network/darwinia-kit
+                  ref: 17064bd5eee5b384581511ec65f9a161aea04176
+            - name: Install Rust nightly-2021-02-25 toolchain
+              uses: actions-rs/toolchain@v1
+              with:
+                  toolchain: nightly-2021-02-25
+                  default: true
+            - name: Build
+              uses: actions-rs/cargo@v1
+              with:
+                  command: build
+                  args: --release --manifest-path ci-bot/Cargo.toml
+              env:
+                  CARGO_INCREMENTAL: 1
+            - name: Compress kit
+              run: |
+                  mv ci-bot avoid-same-name
+                  cp target/release/ci-bot .
+                  tar cf kit.tar.zst ci-bot -I pzstd
+            - name: Upload kit
+              uses: actions/upload-artifact@v2
+              with:
+                  name: kit
+                  path: kit.tar.zst
+    build_node_and_internal_test:
```

Comment:
Personal thoughts: I would add a blank line between every job, especially multiple jobs in a workflow file.

Reply:
Done.